File: app/src/main/java/foocafe/org/foocafe/entities/AccessToken.java
package foocafe.org.foocafe.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class AccessToken implements Serializable {

    @SerializedName("access_token")
    public String access_token;

    @SerializedName("expires_in")
    public String expires_in;

    @SerializedName("user_id")
    public int user_id;

    @SerializedName("token_type")
    public String token_type;

    @SerializedName("refresh_token")
    String refresh_token;
}

File: app/src/main/java/foocafe/org/foocafe/entities/AccessToken.kt
package foocafe.org.foocafe.entities

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class AccessToken : Serializable {

    @SerializedName("access_token")
    var access_token: String? = null

    @SerializedName("expires_in")
    var expires_in: String? = null

    @SerializedName("user_id")
    var user_id: Int = 0

    @SerializedName("token_type")
    var token_type: String? = null

    @SerializedName("refresh_token")
    internal var refresh_token: String? = null
}